src/99-appendices/08-coordinate-systems.md

```diff
@@ -112,10 +116,21 @@ below, the origin is at the AC and the orientation of the axes is RAS. Unless
-| ---------------------------------------------------   | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------   |
+| --------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| FS305                                               | FreeSurfer variant of the MNI305 space                                                                                                                                                                                                                                                                                                                           |
 | MNI152Lin                                           | Also known as ICBM (version with linear coregistration) [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152Lin](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152Lin)                                                                                                                                                                                       |
-| MNI152NLin6\[Sym&#124;Asym\]                        | Also known as ICBM 6th generation (non-linear coregistration). Used by SPM99 -   SPM8 and FSL (MNI152NLin6Sym). [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6)                                                                                                                           |
+| MNI152NLin6\[Sym&#124;Asym\]                        | Also known as ICBM 6th generation (non-linear coregistration). Used by SPM99 - SPM8 and FSL (MNI152NLin6Sym). [http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6](http://www.bic.mni.mcgill.ca/ServicesAtlases/ICBM152NLin6)                                                                                                                             |
+| MNI152NLin6AsymConte69                              | Template based on MNI152NLin6Asym using Conte69 data. Built by Human Connectome Project. [https://github.com/Washington-University/HCPpipelines/tree/master/global/templates](https://github.com/Washington-University/HCPpipelines/tree/master/global/templates)                                                                                                |
```

Comment:
Is this really a new template or it is just the FSL MNI template with the addition of the 1.6mm isotropic sampling (and other resolutions too)? Is there a publication describing the procedure used in building the template in the case it is not a variation of existing ones?

Pinging @edickie, @satra, and also @glasserm and @tbbrown for [a more informed response](https://github.com/Washington-University/HCPpipelines/tree/master/global/templates).

Reply:
I'd need some more context as to what the question is.

Reply:
Sorry, I wasn't clear. The question is whether these templates and related data (https://github.com/Washington-University/HCPpipelines/tree/master/global/templates) are derived from the MNI template packed with FSL, some other template (e.g. the 2009 wave of templates from MNI), or were entirely constructed with the specific purposes of HCP/HCP-Pipelines.

Reply:
The MNI templates are based on FSL's nonlinear template.